src/main/java/com/java_essential/diledsys/homeWork/myList/MyArrayList.java

```diff
@@ -0,0 +1,85 @@
+package com.java_essential.diledsys.homeWork.myList;
+
+import java.util.Arrays;
+
+public class MyArrayList<E> implements MyList {
+
+    private  Object[] array = new Object[1];
```

Comment:
private  E[] array = (E[]) new Object[SIZE];
 The best option for the size of the initial array is 10.